garrysmod/gamemodes/terrortown/gamemode/cl_scoring.lua

```diff
@@ -353,8 +353,14 @@ function CLSCORE:ShowPanel()
    local margin = 15
 
    local dpanel = vgui.Create("DFrame")
-   local w, h = 700, 500
-   dpanel:SetSize(700, 500)
+   if LANG.ActiveLanguage == "français" then

```

Comment:
This is wrong, you can't just hardcode conditions in there.
This is horribly unmodular

Reply:
Add a variable in the translation table/file, not in there

Reply:
No, this should be using surface.GetTextSize instead, but he hasn't updated the PR yet.

Reply:
Yep, even better.
However, doesn't surface.gettextsize has issues with UTF8/16 ?

Reply:
I don't believe so if the font has the ``extended`` key set to true, but I could be mistaken.

Reply:
I'm removing the cl_scoring.lua file from the PR thanks to @Kefta's epic work